pv_site_api/main.py

```diff
@@ -277,6 +278,15 @@ def post_pv_actual(
         )
 
     generation_values_df = pd.DataFrame(generations)
+    site = get_site_by_uuid(session=session, site_uuid=site_uuid)
+    site_capacity_kw = site.capacity_kw
+    exceeded_capacity = generation_values_df[generation_values_df["power_kw"] > site_capacity_kw]
+    if not exceeded_capacity.empty:
```

Comment:
I'd probably go for `len(exceeded_capacity) > 0` as I think its slightly easier to read than two negatives